docs/docs/reference/metaprogramming/macros.md

```diff
@@ -683,21 +646,20 @@ private def sumExpr(argsExpr: Expr[Seq[Int]])(using Quotes): Expr[Int] = argsExp
     // argValues is of type Seq[Int]
     Expr(argValues.sum) // precompute result of sum
   case Varargs(argExprs) => // argExprs is of type Seq[Expr[Int]]
-    val staticSum: Int = argExprs.map(_.value.getOrElse(0)).sum
+    val staticSum: Int = argExprs.map(_.value.getOrElse(0))
```

Comment:
Accidental removal of `.sum`?

Reply:
I've found two other diffs.. I need to update my file.

Reply:
@liufengyun `metaprogramming/macros.md` needs closer revue. Otherwise looks good.